python/servo/command_base.py

```diff
@@ -585,6 +589,11 @@ def build_env(self, is_build=False, test_unit=False):
         if os.path.isdir(tools):
             env["PATH"] = "%s%s%s" % (tools, os.pathsep, env["PATH"])
 
+        if "msvc" in servo.platform.host_triple():
+            # on MSVC, we need some DLLs in the path. They were copied
+            # in to the servo.exe build dir, so just point PATH to that.
+            util.prepend_paths_to_env(env, "PATH", path.dirname(self.get_binary_path(False, False)))
+
```

Comment:
This one is actually dirty hack for unit-tests, which I would like to remove so unit-tests could run independently of `./mach build`. See https://github.com/servo/servo/issues/29815#issuecomment-1602973927